### PR TITLE
Support symfony 3 and firebase jwt token v4.0

### PR DESCRIPTION
**Pull request by monteiro**
@cnam Take a look and merge if you want ;) 

Thanks for creating a JWT service provider for Silex.
- Updated composer.json
- Updated the namespaces in the JWT encoder to new namespace of version 4.0 of https://github.com/firebase/php-jwt

### CODE REVIEW

**Comment by michcald** [comment]
Is this going to be merged any time soon? Thanks

**Comment by Izopi4a** [comment]
this is exactly the reason why people should avoid using other people packages

File: composer.json [hunk 1]
```diff
@@ -21,8 +21,8 @@
     },
     "require": {
         "pimple/pimple": "3.0.*",
-        "firebase/php-jwt": "2.0.0",
-        "symfony/security": "~2.7"
+        "symfony/security": "^3.1",
+        "firebase/php-jwt": "^4.0"
```
Comment (cnam) [inline composer.json]: Please update symfony security to ```~2.8|~3.0```

**Comment by vinyvicente** [comment]
Awaiting response from this pull. 

@monteiro ?

**Comment by monteiro** [comment]
@vinyvicente @cnam let me fix it then.

**Comment by vinyvicente** [comment]
Thanks, @monteiro 

@cnam can help us, please?

**Comment by cnam** [comment]
Hello @vinyvicente , @monteiro.

We release version 2.1.1 with your fix.

**Comment by vinyvicente** [comment]
Thanks, @cnam !

**Pull request merged**